src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml

```diff
@@ -53,11 +53,15 @@ sylius:
     ui:
         catalog_promotion:
             dates_details: This operation is time-consuming! Please consider a 2-10 minutes delay starting from the specified dates. The delay depends on the size of the catalog.
+        channel_pricing_history: 'Channel pricing history'
         channel_pricings: Channel pricings
         product:
             product_not_active_in_channel: The product is not yet activated in this channel.
         original_price_details: Original price - this is the price of the product variant It is displayed as crossed-out in the catalog. It is used as the base for current price calculations. If this value is not defined, Catalog Promotion logic will copy value from Price to Original Price.
         price_details: Price - this is the current price of the product variant displayed in the catalog. It can be modified explicitly by i.e. catalog promotions.
+        price_history: Price history
+        show_history_for_channel_pricing: 'Show history for channel pricing'
+        the_lowest_price_before_the_discount: 'Lowest price before discount'
```

Comment:
Wrong order of keys, and this one should be probably adjusted to the change in plugin